homeassistant/components/zwave_js/discovery.py

```diff
@@ -997,6 +977,25 @@ class ZWaveDiscoverySchema:
         platform=Platform.LIGHT,
         primary_value=SWITCH_MULTILEVEL_CURRENT_VALUE_SCHEMA,
     ),
+    # light for Basic CC
+    ZWaveDiscoverySchema(
+        platform=Platform.LIGHT,
+        hint="Basic",
```

Comment:
I would use Basic CC or basic_cc here to make it more descriptive as now it more looks like a "basic light" instead of "a light based on Basic CC"